### PR TITLE
Update Latest News

### PR DESCRIPTION
**Pull request by shashank3959**
# What does this PR do ?

Adds links to articles on
* NeMo framework on GKE
* Responsible Gen AI using NeMo and Picasso
* NeMo powering Amazon Titan foundation models

### CODE REVIEW

**Review by ericharper — APPROVED** [review]
LGTM. Thanks!

**Comment by titu1994** [comment]
Can you please put it into a droppable menu as shown here, this is too much unrelated things before even beginning to discuss the actual toolkit. If users are interested they can  open the menu. 

For reference from an old version of NeMo - https://github.com/NVIDIA/NeMo/blob/r1.20.0/PUBLICATIONS.md

File: README.rst [hunk 1]
```diff
@@ -41,17 +41,43 @@
...
+    <summary><b>Large Language Models and Multimodal</b></summary>
+        <details open>
+          <summary><a href="https://cloud.google.com/blog/products/compute/gke-and-nvidia-nemo-framework-to-train-generative-ai-models">Accelerate your generative AI journey with NVIDIA NeMo framework on GKE</a> (2024/03/16) </summary>
 
-NeMo Framework has been updated with state-of-the-art features,
-such as FSDP, Mixture-of-Experts, and RLHF with TensorRT-LLM to provide speedups up to 4.2x for Llama-2 pre-training on H200.
-**All of these features will be available in an upcoming release.**
+          An end-to-end walkthrough to train generative AI models on the Google Kubernetes Engine (GKE) using the NVIDIA NeMo Framework is available at https://github.com/GoogleCloudPlatform/nvidia-nemo-on-gke. The walkthrough includes detailed instructions on how to set up a Google Cloud Project and pre-train a GPT model using the NeMo Framework.
+          <br><br>
+        </details>
```
Comment (ericharper) [inline README.rst:53]: This isn't folding properly. It still shows before expanding

**Review by ericharper — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**